API.Models/Models.Persons/PersonDetail.cs

```diff
@@ -0,0 +1,22 @@
+﻿using API.Data;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace API.Models.Models.Persons
+{
+    public class PersonDetail
+    {
+        public int ID { get; set; }
+        public string Name { get; set; }
+        public string Title { get; set; }
+        public int Age { get; set; }
+        public string Occupation { get; set; }
+        //Add Location
+        [ForeignKey(nameof(Place))]
+        public virtual Place Place { get; set; }
```

Comment:
I don't think you need the ForeignKey annotation here
```suggestion
        //Add Location
        public Place Place { get; set; }
```